internal/generator/fluentd/pipeline_to_output.go

```diff
@@ -81,7 +81,8 @@ func PipelineToOutputs(spec *logging.ClusterLogForwarderSpec, op Options) []Elem
 				Match{
 					MatchTags: "**",
 					MatchElement: Copy{
-						Stores: CopyToLabels(helpers.LabelNames(p.OutputRefs)),
+						DeepCopy: true,
```

Comment:
instead of always setting deep copy to true, we can add it selectively by detecting if we need deep copying. e.g. using syslog and ES together

Reply:
agree